### PR TITLE
Apply common project tweaks

### PR DESCRIPTION
**Pull request by vweevers**
Squash on merge. Ref Level/community#29.

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+| **Tim Kuijsten**           | [**@timkuijsten**](https://github.com/timkuijsten)     | [**@timkuijsten@mastodon.social**](https://mastodon.social/@timkuijsten) |
+| **Michael Nisi**           | [**@michaelnisi**](https://github.com/michaelnisi)     |                                                                          |
+| **Sharvil Nanavati**       | [**@sharvil**](https://github.com/sharvil)             |                                                                          |
+| **Thorsten Lorenz**        | [**@thlorenz**](https://github.com/thlorenz)           |                                                                          |
+| **Andrew Kelley**          | [**@andrewrk**](https://github.com/andrewrk)           |                                                                          |
+| **Adam Bliss**             | [**@abliss**](https://github.com/abliss)               |                                                                          |
+| **flames of love**         |                                                        |                                                                          |
```
Comment (vweevers) [inline CONTRIBUTORS.md]: hehe what

File: package.json [hunk 2]
```diff
@@ -41,17 +43,22 @@
...
+    "community": "level-community",
+    "ignore": [
+      "README.md"
+    ]
   },
-  "license": "MIT",
   "gypfile": true,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/level/leveldown.git"
```
Comment (vweevers) [inline package.json]:
```suggestion
    "url": "https://github.com/Level/leveldown.git"
```

File: package.json [hunk 3]
```diff
@@ -41,17 +43,22 @@
...
+  },
+  "homepage": "https://github.com/level/leveldown",
```
Comment (vweevers) [inline package.json]:
```suggestion
  "homepage": "https://github.com/Level/leveldown",
```

File: CONTRIBUTORS.md (hunk 1 above)
Comment (ralphtheninja) [reply CONTRIBUTORS.md]: :burn: :heart:

File: README.md [hunk 4]
```diff
@@ -374,16 +374,10 @@ $ git submodule update --init --recursive
 7. Add changelog to the GitHub release
 8. Finally: `npm publish`
 
-### Windows
```
Comment (ralphtheninja) [inline README.md]: We just scrap this?

Comment (vweevers) [reply README.md]: Moved to the `License` section

**Pull request merged**